GDJS/Runtime/pixi-renderers/CustomRuntimeObject2DPixiRenderer.ts

```diff
@@ -80,8 +80,10 @@ namespace gdjs {
         this._object.getY() + this._pixiContainer.pivot.y * Math.abs(scaleY);
 
       this._pixiContainer.rotation = gdjs.toRad(this._object.angle);
-      this._pixiContainer.scale.x = scaleX;
-      this._pixiContainer.scale.y = scaleY;
+      this._pixiContainer.scale.x =
+        scaleX * (this._object.isFlippedX() ? -1 : 1);
```

Comment:
this was a bug in the runtime, custom objects (like a button) could not be flipped

Reply:
@D8H tagging you so that you know about this.